app/src/main/res/layout/fragment_edit_drug.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#66CCFF"
```

Comment:
Add `#66CCFF` to the colors.xml with a meaningful name.

Reply:
Resource extracted.
81223eb832dd8f9d7ad51806b390a9f997067dd5